vertical-pod-autoscaler/pkg/utils/metrics/quality/quality.go

```diff
@@ -57,20 +57,36 @@ var (
 			Help:      "Count of usage samples when a recommendation should be present but is missing",
 		}, []string{"update_mode", "resource", "is_oom"},
 	)
-	cpuUsageRecommendationDiff = prometheus.NewHistogramVec(
+	cpuRecommendationOverUsageDiff = prometheus.NewHistogramVec(
 		prometheus.HistogramOpts{
 			Namespace: metricsNamespace,
-			Name:      "cpu_usage_recommendation_diffs_cores",
-			Help:      "Absolute diffs between usage and recommendation for CPU",
-			Buckets:   prependNegativeBuckets(cpuBuckets),
+			Name:      "cpu_recommendation_over_usage_diffs_cores",
+			Help:      "Absolute diffs between usage and recommendation for CPU when recommendation > usage",
+			Buckets:   cpuBuckets,
+		}, []string{"update_mode", "recommendation_missing"},
+	)
+	memoryRecommendationOverUsageDiff = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: metricsNamespace,
+			Name:      "mem_recommendation_over_usage_diffs_bytes",
+			Help:      "Absolute diffs between usage and recommendation for memory when recommendation > usage",
+			Buckets:   memoryBuckets,
+		}, []string{"update_mode", "recommendation_missing", "is_oom"},
```

Comment:
I don't think we can have "recommendation_missing" set to true for this particular metric (and the one for CPU) but I'd still like to keep it to have the labels in line with "LowerOrEquals" metric. WDYT?

Reply:
I agree